cms/envs/common.py

```diff
@@ -323,7 +344,11 @@
 ############################# SOCIAL MEDIA SHARING #############################
 SOCIAL_SHARING_SETTINGS = {
     # Note: Ensure 'CUSTOM_COURSE_URLS' has a matching value in lms/envs/common.py
-    'CUSTOM_COURSE_URLS': False
+    'CUSTOM_COURSE_URLS': False,
+    # 'DASHBOARD_FACEBOOK': False,
```

Comment:
Same thing here - if they're set in configuration can we add them here or were they commented because they cause the tests to fail (Perhaps this is just a WIP?)